all/plugin-dialects/README.md

```diff
@@ -2,10 +2,178 @@
 
 FAW plugin for partitioning the corpus into dialects.
 
+
+## Motivation
+
+The dialects wizard is a tool for finding mutually disjoint features that cover
+the corpus, or a subset of it. Like the clustering plugin, it is intended for
+root cause analysis, i.e. finding the relationships between interesting
+features. Unlike the clustering plugin, which primarily deals only with pairs
+of features (making its name a misnomer), the dialect wizard facilitates dealing
```

Comment:
Hey now, documenting that a name is a misnomer? (it's a reasonable thing to do)

Reply:
Thinking about it a little more, it's not _entirely_ a misnomer. I'll remove this parenthetical.